src/FAI3_frontend/src/pages/Leaderboard/LeaderboardTable.tsx

```diff
@@ -92,15 +93,16 @@ export default function LeaderboardTable() {
         );
       },
       cell: ({ row }: any) => {
-        const metrics = row.original.metrics;
+        const model_data = row.original.model_type['Classifier'] as ClassifierModelData; 
+        const metrics: Metrics = model_data.metrics;
         const cellValue = Number(metrics.average_metrics.statistical_parity_difference[0]);
 
         return isNaN(cellValue) ? null : (
             <div
               className={`ml-4 w-fit py-0.5 px-2 rounded-[10px]
-                                ${metrics[0] < 0.1 && metrics[0] > -0.1
+                                ${cellValue < 0.1 && cellValue > -0.1
                   ? `text-[#007F00] bg-[#CDFFCD80] bg-opacity-50`
-                  : metrics[0] > 0.4 || metrics[0] < -0.4
+                  : cellValue > 0.4 || cellValue < -0.4
```

Comment:
I changed these `metrics[index]` to cellValue because metrics is not an array. Please let me know if this change maintains the original purpose of this code.